x-pack/plugins/enterprise_search/public/applications/enterprise_search/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const PRODUCT_SELECTOR_CALLOUT_HEADING = i18n.translate(
+  'xpack.enterpriseSearch.productSelectorCalloutHeading',
```

Comment:
[Optional] Not a huge deal/blocker, but Kibana recommends `Title` suffixes for heading text: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

```suggestion
  'xpack.enterpriseSearch.productSelectorCalloutTitle',
```